lib/tessel/deployment/rust.js

```diff
@@ -33,32 +37,134 @@ var exportables = {
         program
       };
     },
-    shell: () => {
+    shell: (opts) => {
       return tags.stripIndent `
         #!/bin/sh
-        exec /app/remote-script/rust_executable
+        exec /app/remote-script/${opts.resolvedEntryPoint}
```

Comment:
This no longer needs to be changed. In https://github.com/tessel/t2-cli/commit/e334d8eb23dd4eb902fe0b82ba1babd8e8174871, I opted to simply accept the flash path as-is, since it's been like that for since https://github.com/tessel/t2-cli/commit/4eb17770a20e68ad699b88e47f035ea03b41558f and it's really not a problem.